From 623fbfa8799e80facf17651735bf467588cb4809 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Sat, 19 Jun 2021 11:22:24 +0200
Subject: [PATCH 5/6] Revert "Makefile: Turn off loop vectorization for GCC -O3
 optimization level"

This reverts commit bbd05a1b94218ad6cbd76d9e9540af4171ab85f7.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Makefile b/Makefile
index 4a408fd30..f02860856 100644
--- a/Makefile
+++ b/Makefile
@@ -737,9 +737,6 @@ ifdef CONFIG_CC_OPTIMIZE_FOR_PERFORMANCE
 KBUILD_CFLAGS += -O2
 else ifdef CONFIG_CC_OPTIMIZE_FOR_PERFORMANCE_O3
 KBUILD_CFLAGS += -O3
-ifdef CONFIG_CC_IS_GCC
-KBUILD_CFLAGS += -fno-tree-loop-vectorize
-endif
 else ifdef CONFIG_CC_OPTIMIZE_FOR_SIZE
 KBUILD_CFLAGS += -Os
 else ifdef CONFIG_CC_OPTIMIZE_BASAL
-- 
2.32.0

